sleepecg/test/test_config.py

```diff
@@ -0,0 +1,54 @@
+# Authors: Florian Hofer
+#
+# License: BSD (3-clause)
+
+"""Tests for the configuration interface."""
+
+from pathlib import Path
+
+import pytest
+
+import sleepecg.config
+from sleepecg import get_config, set_config
+
+_TESTCONFIG_PATH = '~/.sleepecg/testdata/testconfig.yml'
```

Comment:
I guess it's OK to test in the user directory, but we should make sure that any directory or file that gets created during testing is always deleted (even if a test fails). I'm not sure if this is currently guaranteed. I thought about using the `tmp_path` fixture for that, but then we can't test the default location (which is in the user's home) - but maybe we don't have to?

Reply:
Yes, good idea! I agree, the default location should not be necessary